src/main/groovy/jenkins/automation/utils/CheckmarxSecurityJobBuilder.groovy

```diff
@@ -0,0 +1,63 @@
+package jenkins.automation.utils
+
+import javaposse.jobdsl.dsl.DslFactory
+import javaposse.jobdsl.dsl.Job
+
+
+class CheckmarxSecurityJobBuilder {
+
+    String name
+    String description
+    String baseUrl
+    String scanRepo
+    String checkmarxComment
+    Boolean vulnerabilityThresholdEnabled
+    String highThreshold
+    String mediumThreshold
+    String lowThreshold
+
+    Job build(DslFactory factory){
+        factory.job(name){
+            it.description this.description
+            
+            scm {
+                git {
+                    remote {
+                        name('Repo to Scan')
+                        url(scanRepo)
+                    }
+                    
+                    branch('*/master')
+                    wipeOutWorkspace(true)
+                }
+            }
+
+            configure { project -> 
+                project / builders / 'com.checkmarx.jenkins.CxScanBuilder' (plugin:"checkmarx@7.2.3-31") {
```

Comment:
I removed the version and it ran without problems, @imuchnik any reason we should keep the plugin version?